### PR TITLE
add scenarios, add registration implementation, fix add property

### PR DESCRIPTION
**Pull request by ewakotanska**

### CODE REVIEW

**Review by wgierczak — CHANGES_REQUESTED** [review]

File: cypress/integration/dashboard_manager.feature [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  Scenario: Manager account can view and edit contact information
+
+  @notimplemented
+  Scenario: Manager account can view and edit property details
+
+  @notimplemented
+  Scenario: Manager account can see and edit customer care tab
+
+  @notimplemented
+  Scenario: Manager account can see list of bookings, see details of booking including card details
```
[the remark this answers is not in the document]
Comment (ewakotanska) [reply cypress/integration/dashboard_manager.feature:62]: its the same in selenium suite. The task was to align those two. But we can think about it.

File: cypress/support/index.js [hunk 2]
```diff
@@ -16,5 +16,7 @@
 // Import commands.js using ES2015 syntax:
 import './commands'
 
+Cypress.new_email = () => `e2e+test${Math.floor(Math.random() * 1000000000)}@bidroom.com`
+Cypress.new_property_name = () => `TestProperty${Math.floor(Math.random() * 1000000000)}`
```
[the remark this answers is not in the document]
Comment (ewakotanska) [reply cypress/support/index.js:20]: I will create task for it

File: cypress/support/propertiesPage.js [hunk 3]
```diff
@@ -114,12 +119,23 @@ export class PropertiesPage {
...
+            return $popup.is(':visible')
+            
+        })
     }
 
     check_if_newly_added_property_is_visible() {
-        cy.get("#searchValue").type(test_property.name).click()
-        return cy.get("[data-property-name]").contains(test_property.name)
+        cy.get("#searchValue").type(this.new_property_name).click()
+        return cy.get("[data-property-name]", { timeout: 10000 }).contains(this.new_property_name)
```
[the remark this answers is not in the document]
Comment (ewakotanska) [reply cypress/support/propertiesPage.js:138]: the problem with timeout was only here, but we can discuss it

File: cypress/support/registrationPage.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import new_user from '../fixtures/new_user.json'
+export class RegistrationPage {
+
+    constructor(new_email){
+        this.new_email = Cypress.new_email(); 
+    }
```
[the remark this answers is not in the document]
Comment (ewakotanska) [reply cypress/support/registrationPage.js:6]: there is no user class. Test data is taken from jsons

File: cypress/integration/dashboard_manager.feature (hunk 1 above)
[the remark this answers is not in the document]
Comment (wgierczak) [reply cypress/integration/dashboard_manager.feature:62]: ok, lets think about it and make separate task if needed

**Review by wgierczak — APPROVED** [review]

**Pull request merged**